### ISSUE TITLE
Chunk Names Bug

### ISSUE THREAD

**Issue by brandonferens** [JeffreyWay/laravel-mix#194]
I noticed this output under chunk names
```
           /js/team/index.js    3.66 kB       0  [emitted]  /js/team/index
                /css/app.css    0 bytes       0  [emitted]  /js/team/index
/css/vendor/querybuilder.css      17 kB       0  [emitted]  /js/team/index
           mix-manifest.json  146 bytes          [emitted]
```
I'm assuming the chunk names should be respective of their filenames, not the first one. I believe this is just an issue with css. Any combination I attempt, the last js chunk name is use for the css ones.

**Comment by JeffreyWay** [comment]
Can you paste your webpack.mix.js file?

**Comment by brandonferens** [comment]
With the commit that fixed #193, this seems to not be an issue.

**Comment by JeffreyWay** [comment]
Yeah, sorry, should have clarified that.

Sorry for the trouble, but thanks for the help!
